app/components/avatar/avatar.css

```diff
@@ -1,12 +1,20 @@
 .Avatar {
     display: inline-block;
     box-sizing: content-box;
-    width: calc(3 * var(--spacing-unit));
```

Comment:
Why don’t we keep one of them as the default variant and only add a single modifier? I think having a default of 2 * spacing-unit, and then a large variant would be fine.